apps/studio/src/lib/db/clients/sqlite.js

```diff
@@ -252,7 +252,7 @@ export async function applyChanges(conn, changes) {
 
 export async function updateValues(cli, updates) {
   const commands = updates.map(update => {
-    const params = [update.value];
+    const params = [_.isBoolean(update.value) ? _.toInteger(update.value) : update.value];
```

Comment:
Could you add an integration test for this to tests/integration/lib/db/sqlite? You should verify that the test FAILS without this fix, then verify that this patch fixes the test.

Reply:
Make sure to also test setting the value to `null` to make sure no one breaks this in the future, for sure we want to make sure if someone sets the value to `null` it doesn't turn to `false` by mistake.

Reply:
Hi @rathboma , I've just added the tests for this. Also it's probably worth to mention that if you create a table that contains `boolean` and `bool` data types:

```sql
CREATE TABLE boolean_table (
	id integer PRIMARY KEY AUTOINCREMENT,
   	flag1 bool,
	flag2 boolean
);
```

Beekeeper would should show true/false autocomplete on the `bool` column, but not on the `boolean` column, which would allow users to insert any data type other than 0 and 1.

Reply:
oh, that seems like a mistake, can you fix that also?

Reply:
No problem, I'll add that in this PR soon :)

Reply:
done